mk/mk.deps

```diff
@@ -663,6 +663,7 @@ test/opmlurlreader.o: test/opmlurlreader.cpp include/opmlurlreader.h \
  include/configactionhandler.h include/urlreader.h 3rd-party/catch.hpp \
  test/test-helpers.h include/utils.h include/logger.h config.h \
  include/strprintf.h
+test/download.o: test/download.cpp include/download.h
```

Comment:
This is missing `test/test-helpers.h`. Note that this file is updated by `make deps`, so you can just run that and commit the result.

We always forget to re-run that command, so don't worry if there are some unrelated changes—just commit the whole file.

Reply:
Apparently the make target is called slightly differently: `make depslist`

Reply:
Ah, yes, sorry about that—wrote the command from memory without checking.